src/engine/renderer/tr_shade.cpp

```diff
@@ -1971,6 +2139,11 @@ void Render_heatHaze( shaderStage_t *pStage )
 
 	GLimp_LogComment( "--- Render_heatHaze ---\n" );
 
+	// FIXME
```

Comment:
HeatHaze needs either special handling or to be split into 2 distinct stages since Render_heatHaze does 2 drawcalls, while this system as a whole assumes 1 drawcall per stage. Perhaps it's better to remove this comment from here since it's explained elsewhere in the code.

Reply:
Replaced with a different comment that has an actual explanation.